src/components/molecules/BurgerMenu.tsx

```diff
@@ -36,6 +37,8 @@ const MenuItem = ({icon, title, onClick}: MenuItemProps) => {
 const BurgerMenu = ({visible, onHide}: BurgerMenuProps): JSX.Element => {
     const navigate = useNavigate()
     const {t: translate} = useTranslation(namespaces.components.burgerMenu);
+    const menuTitle = 'BALPA'
```

Comment:
I would add the menu title to the i18n translations, even if we don't need to translate the title, I think is better to have all the texts in one place, what do you think?